docs/contracts.rst

```diff
@@ -56,14 +56,15 @@ This means that cyclic creation dependencies are impossible.
         // creator and the assigned name.
         constructor(bytes32 _name) public {
             // State variables are accessed via their name
-            // and not via e.g. this.owner. This also applies
+            // and not via e.g., `this.owner`. This also applies
             // to functions and especially in the constructors,
-            // you can only call them like that ("internally"),
+            // you can only call them "internally",
```

Comment:
At this point, people do not yet know what "calling internally" means, so I think we should somehow keep the connection. I see the problem, though that "like this" does not really clarify what "this" is :)

Reply:
@chriseth How about "from functions declared in this code", "you can only call functions declared in this contract code", or something like that? I'd finesse that text but I _think_ that encompasses what you mean?

Reply:
Perhaps: you should not call them via `this.`. More on that later.

Reply:
@axic and @chriseth how about this? I wish you could create links in code comments, but you can't :(